### DISCUSSION TITLE
Always show enabled sublists?

### DISCUSSION THREAD

**Discussion by MasterKia** [uBlockOrigin/uBlock-issues#3045] [Ideas]
I don't have a convincing argument as to why it's better to be like this, but I think it would provide a better quick glance to see which lists are enabled without using too much real estate by the disabled list.

Currently it's like:
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/17685483/e8ad260c-5f50-44bd-b884-6cc669a54b77)

My suggestion:
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/17685483/da2b3fdd-105c-4d52-a447-693f6b60fcc0)

**Comment by krystian3w** [comment]
Looks as bug on second screenshot.

**Comment by peace2000** [comment]
Imo, showing selected sublists is a good idea, but hiding their unselected siblings isn't as it gives an idea that they don't exist.

**Reply by u-RraaLL** [reply]
The expander arrow on the right is closed. The only difference from the current behavior is that the selected sublists wouldn't be hidden.

Then again, not much point if somebody wanted to enable all sublists anyway. <= They wouldn't be able to collapse the section.
